api/analysis.go

```diff
@@ -1201,8 +1203,10 @@ func (h AnalysisHandler) AppIssueReports(ctx *gin.Context) {
 	for i := range list {
 		m := list[i]
 		r := &IssueReport{
-			Files:       m.Files,
-			Description: m.Description,
+			Files: m.Files,
+			// Append Incident Message to Description to provide more information on Issue detail
+			// (workaround until Analyzer output Violation struct gets updated to provide better structured data)
+			Description: fmt.Sprintf("%s\n\n%s", m.Description, m.Message),
```

Comment:
I wonder if this comment is accurate or necessary.  What is the anticipated change in the analyzer data?
This report is a denormalization already.

Reply:
I think the main change should be adding Message field to the Violation/Issue, proposal that was discussed mainly with Juanma, Ramon and Shawn is https://github.com/konveyor/analyzer-lsp/discussions/452#discussioncomment-8055352

When implementing the proposal, the Violation struct in analyzer-lsp would be changed (added fields and its population) and components using it as dependency including builder in tackle2-addon-analyzer, likely also Hub and UI will need adapt to this struct change too. That update in Hub should remove change introduced in this PR.